src/applications/personalization/dashboard/components/cerner-widgets.js

```diff
@@ -0,0 +1,128 @@
+
+// Returns an AlertBox to present the user with info about working with the
+// Cerner facility they are enrolled at. Props allow you to edit a small amount
+// of the content that is rendered in the AlertBox.
+const CernerAlertBox = ({
+  ctaButtonText,
+  ctaButtonUrl,
+  ctaText,
+  facilityNames,
+}) => {
+  // Helper component that takes an array of facility names and a separator string and returns some JSX to style the list of facility names.
+  const FacilityList = ({ facilities, separator }) => {
+    const newArray = [];
+    // first make an array that alternates between a facility name and the
+    // separator word wrapped in a <span> for styling purposes
+    facilities.forEach(el => {
+      newArray.push(el);
+      newArray.push(
+        <span className="vads-u-font-weight--normal vads-u-font-size--base">
+          {separator}
+        </span>,
+      );
+    });
+    // we don't need the last separator in the array
+    newArray.pop();
+    // Then map over the array we just made, converting it to JSX
+    return (
+      <>
+        {newArray.map((el, i) => {
+          return <Fragment key={i}>{el}</Fragment>;
+        })}
+      </>
+    );
+  };
+
+  return (
+    <AlertBox
+      status="warning"
+      headline="Your VA health care team may be using our new My VA Health portal"
+    >
+      <h3>Our records show you’re registered at:</h3>
+      <h4>
+        <FacilityList facilities={facilityNames} separator=" and " />
+        <span className="vads-u-font-weight--normal vads-u-font-size--base">
+          {' '}
+          (Using My VA Health)
+        </span>
+      </h4>
+      <p>
+        Please choose a health management portal below, depending on the
+        facility for your appointment. You may need to disable your browser’s
+        pop-up blocker to open the portal. If you’re prompted to sign in again,
+        use the same account you used to sign in on VA.gov.
+      </p>
+      <h3>{ctaText}</h3>
+      <h4>
+        <FacilityList facilities={facilityNames} separator=" or " />
+      </h4>
+      <a
+        href={
+          environment.isProduction()
+            ? 'https://patientportal.myhealth.va.gov/'
+            : 'https://ehrm-va-test.patientportal.us.healtheintent.com/'
+        }
+        type="button"
+        className="usa-button-primary"
+        rel="noopener noreferrer"
+        target="_blank"
+      >
+        Go to My VA Health
+      </a>
+      <h4>Another VA health facility</h4>
+      <a href={ctaButtonUrl} type="button" className="usa-button-secondary">
+        {ctaButtonText}
+      </a>
+    </AlertBox>
+  );
+};
+
+export const CernerScheduleAnAppointmentWidget = ({ facilityNames }) => (
+  <div data-testid="cerner-appointment-widget">
+    <h3>View, schedule, or cancel an appointment</h3>
+    <CernerAlertBox
+      ctaText="Manage appointments at:"
+      ctaButtonText="Go to the VA appointments tool"
+      ctaButtonUrl="/health-care/schedule-view-va-appointments/"
```

Comment:
Can we use `appointmentsToolLink` from `src/platform/utilities/cerner/index.js` here? Not sure since it has `/appointments` appended

Reply:
I'm not sure. It seems likely, based on how the PW team is using that URL, but this is the URL that's being used by the current non-Cerner appt widget.

@kelsonic do you happen to know if `/health-care/schedule-view-va-appointments/appointments` and `/health-care/schedule-view-va-appointments/` are essentially interchangeable?

Reply:
I think those are 2 separate pages, never have gone to `/health-care/schedule-view-va-appointments/appointments` tbh

Reply:
Thanks. I'll leave this as is since that's the link we've been using.